src/containers/shared/utils.js

```diff
@@ -438,3 +438,41 @@ export const fetchMetrics = () =>
 
 export const removeRoutes = (routes, ...routesToRemove) =>
   routes.filter((route) => !routesToRemove.includes(route.title))
+
+export const formatAsset = (asset) =>
+  typeof asset === 'string'
+    ? { currency: 'XRP' }
+    : {
+        currency: asset.currency,
+        issuer: asset.issuer,
+      }
+
+export const formatBalance = (asset) => {
+  const drop = 1000000
+  return typeof asset === 'string'
+    ? { currency: 'XRP', amount: Number(asset) / drop }
+    : {
+        currency: asset.currency,
+        amount: Number(asset.value),
+        issuer: asset.issuer,
+      }
+}
+
+export const localizeBalance = (balance, language) => {
```

Comment:
Why are only those currencies to display the currency code?

Reply:
We actually dont display currency code for these currencies. The reasoning is that localizeNumber will add the icons for these main 3 currencies but not for others so in all other cases we include the standard shorthand for the currencies i.e. 'SOLO'. So for xrp it would look like'[xrp icon]400' instead of 'XRP [xrp icon]400' 

Let me know if you want me to remove it.